Libraries/ActionSheetIOS/ActionSheetIOS.js

```diff
@@ -25,6 +26,18 @@ var ActionSheetIOS = {
       typeof callback === 'function',
       'Must provide a valid callback'
     );
+
+    if (options.tintColor) {
+      invariant(
+        typeof options.tintColor === 'string' && options.tintColor !== '',
```

Comment:
Remove `options.tintColor !== ''` -- it's guaranteed to be true since you checked `if (options.tintColor)`